contracts/extensions/sequential-governance/CoreGovernance.sol

```diff
@@ -240,6 +273,31 @@ abstract contract CoreGovernance is SignatureConsumer, VoteStatusConsumer {
     }
   }
 
+  /**
+   * @dev When the contract is on Ronin chain, checks whether the proposal is expired and delete it if expired.
+   */
+  function _tryDeleteExpiredVotingRound(ProposalVote storage _proposalVote) private returns (bool _isExpired) {
```

Comment:
Please consider exposing this function (in `RoninGovernanceAdmin`) to support clearing expired votes without create new proposals